tower-service-util/src/call_all/common.rs

```diff
@@ -0,0 +1,99 @@
+use super::Error;
+use futures::{try_ready, Async, Future, Poll, Stream};
+use tower_service::Service;
+
+/// TODO: Dox
+#[derive(Debug)]
+pub(crate) struct CallAll<Svc, S, Q> {
+    service: Svc,
+    stream: S,
+    queue: Q,
+    eof: bool,
+}
+
+pub(crate) trait Queue<T: Future> {
```

Comment:
`Queue` is a little weird of a name for this, since `FutureUnordered` isn't a `Queue` (i.e., FIFO).

Reply:
I renamed -> `Drive` 🤷‍♂️